README.md

```diff
@@ -141,6 +141,16 @@ For example, for `ARCH=arm64` and `VERSION=v1-0-0` this should yield:
 arn:aws:lambda:us-east-1:247286868737:layer:firetail-extension-x86_64-v1-0-4:1
```

Comment:
update to arn:aws:lambda:us-east-1:247286868737:layer:firetail-extension-x86_64-v1-0-5:1